modules/jjwg_Maps/language/en_us.lang.php

```diff
@@ -143,6 +143,7 @@
 $mod_strings['LBL_CONFIG_GEOCODING_API_URL_DESC'] = 'The URL of the Google Maps API V3 or Proxy';
 $mod_strings['LBL_CONFIG_GEOCODING_API_SECRET_TITLE'] = 'Secret Phrase for Proxy:';
 $mod_strings['LBL_CONFIG_GEOCODING_API_SECRET_DESC'] = 'The Secret Phrase to be used with the Proxy MD5 comparison.';
+$mod_strings['LBL_CONFIG_GEOCODING_API_SECRET_GET_A_KEY'] = 'Get a free Google Maps Geocoding API key (to geocode 2500 addresses per day for free).';
```

Comment:
Better not include the free limits details `(to geocode 2500 addresses per day for free)` as this can be changed by google in the future and it will require new translations.